code/nrf-connect/samples/ble/src/main.c

```diff
@@ -34,6 +34,17 @@ static int prst_loop(prst_sensors_t *sensors) {
 int main(void) {
   __ASSERT(!prst_init(), "Error in prst_init()");
   prst_led_flash(2);
+
+  if (IS_ENABLED(CONFIG_PRST_BLE_ENCODING_BTHOME_V2)) {
```

Comment:
This will silently output nothing when new encoding schemes are introduced. We have this information at compile time, we can also avoid paying the binary size cost for all these strings. 

```C
static void dump_config() {

#if ENABLED(CONFIG_PRST_BLE_ENCODING_BTHOME_V2)
// Only capitalize first letter.
  LOG_INFO("Payload encoding: BTHOME_V2");
#else if ...
...
#else
#error "Unhandled encoding in dump_config()"
#endif

  LOG_INF("Sleep duration: %d ms", CONFIG_PRST_SLEEP_DURATION_MSEC);
}

int main(void) {
  ...

  dump_config();

  ...
}
```